src/odoc.ml

```diff
@@ -87,9 +87,10 @@ module Gen (S : sig val sctx : SC.t end) = struct
     let setup_deps m files = SC.add_alias_deps sctx (alias m) files
   end
 
-  let odoc = lazy (
```

Comment:
why is lazy removed here? The same arrow should work in each case.

Reply:
Hmm I thought it was for memoizing, but it's just to avoid computing it at all if not necessary. I'll add it back. I had trouble with the types though, since I was relying on the polymorphism in argument type (this is why I made it a function). I'll try to rework that part.